projects/vanilla/src/app/app-config.service.ts

```diff
@@ -26,6 +29,22 @@ export class AppConfigService {
           this.globalService.startListening();
         }
       });
+
+      // Merge the overriden translations inside the currentLocale messages
+      let messages: any;
+      let locale: string;
+      let translations: any;
```

Comment:
`translation` is never assigned, so it is always undefined, no?

Reply:
you still don't need this variable I think

Reply:
You're sure? I feel like if we change the language it's useful to have it in the `intlService.events` subscription